### ISSUE TITLE
rcCommand and gyro graphs do not match

### ISSUE THREAD

**Issue by IharpI** [betaflight/blackbox-log-viewer#28]
The numbers in the legend match but the graphs do not although configured the same way.
This way it is impossible to compare the desired rate with the rate of the copter.

![graph mismatch settings](https://cloud.githubusercontent.com/assets/6151274/18350853/25842b7c-75d8-11e6-9a34-9474dbf499f2.png)
![graph mismatch](https://cloud.githubusercontent.com/assets/6151274/18350854/25865b22-75d8-11e6-9953-2a909faaaade.png)

**Comment by GaryKeeble** [comment]
Hi,

The scaling of rcCommand[] graph is always +/-500 (raw values), the value shown in brackets on the legend is converted from this value to the real deg/s so that you can see what it represents. This is how the original log viewer worked and has been left that way for compatibility. (which makes it difficult to compare gyro and rcCommand traces as you've pointed out).

However, I have added a group of pens called rcCommand**s**[] which are shown on the pen selection drop-down at the very end.... these are scaled in deg/s so you can use them to compare rcCommands with gyro and if you have perfect PID's, they should exactly overlay each other. These are the ones I use all the time.

<img width="1215" alt="rccommands" src="https://cloud.githubusercontent.com/assets/7362663/18351430/103ac8f8-75d2-11e6-81b9-394823ef4a2c.png">

**Comment by borisbstyle** [comment]
Maybe rcCommands should be called commandedRate, angleRate or setpointRate?

**Comment by IharpI** [comment]
Thanks for the reply and sorry that I did not find the rcCommand**s**
I just checked the CF viewer and in the latest version the rcCommand matches the behavior of rcCommand**s** in BF log viewer. At least there the rcCommand matches the trace of the gyro pretty good.
Why not just rename rcCommand to something like rcCommandRaw and rcCommand**s** to rcCommand?

**Comment by GaryKeeble** [comment]
@borisbstyle Actually, I couldn't think of of a good name for them when I created them (I added 's' for 'scaled', lazy and a naming cop out for sure :-) ). As a old-school controls engineer, I like your term 'setpointRate' as that pretty much says what they are; we could go with that?

@IharpI I didn't realise the original CF viewer had any of the deg/s calculations in it; thanks for letting me know. I think you and Boris are on the same wavelength and a name change is in order.
